Create regular participles in Dutch

agnieszkaszuba opened the pull request:
## Summary

This PR can be summarized in the following changelog entry:

Creates participles of regular Dutch verbs. 

## Test instructions

This PR can be tested by following these steps:

Add more examples to the specs. The following should be covered:

* Participles that take the -_t_ or -_d_ suffix (or no suffix if the stem already ends in -_t_ or -_d_) and that take the _ge_ prefix, e.g. _toon_ -> _getoond_, with an optional separable prefix (e.g. _om_) that comes before the _ge_ prefix, e.g. _omdraai_ -> _omgedraaid_
* Participles of compound verbs with an inseparable prefix (e.g., _be_) that take the _-d_ suffix, e.g. _behandel_ -> _behandeld_. Participles of inseparable compound verbs with the _-t_ suffix or with no suffix are the same as other verb forms (1st or 3d person singular) so they are already covered.
* Exception verbs that have a separable prefix which in that specific verb is inseparable (e.g., _omringd_) are also covered. The exception list can be found in the morphology data file under `inseparableVerbs`.

The following are NOT covered:
- participles of verbs that are on verb exception lists, e.g. participles of strong verbs (these will be covered later)
- participles with uncommon separable prefixes, e.g. _zwart_ as in _zwartgemaakt_. The list of separable prefixes that are currently taken into account can be found in the morphology data under `compoundVerbsPrefixes` -> `separable`.


## Quality assurance

* [ ] I have tested this code to the best of my abilities
* [ ] I have added unittests to verify the code works as intended

Fixes #389

hansjovis commented on packages/yoastseo/src/morphology/dutch/generateParticipleForm.js:
```diff
@@ -0,0 +1,153 @@
+ */
+const generateInseparableExceptionParticipleForm = function( stemmedWord, inseparableExceptions ) {
+	if ( inseparableExceptions.includes( stemmedWord ) ) {
+		return addParticipleAffixes( stemmedWord, "d" );
```
I would simplify it to this:
```suggestion
		return stemmedWord + "d";
```
This communicates directly to the reader of the code that we add the `d` suffix to the end of the stemmed word. Wrapping it into a separate function adds a layer of indirection, in my opinion.

hansjovis commented on packages/yoastseo/src/morphology/dutch/generateParticipleForm.js:
```diff
@@ -0,0 +1,153 @@
+
+			const prefix = morphologyDataVerbs.participleAffixes.prefix;
+
+			return addParticipleAffixes( stemmedWordWithoutPrefix, suffix, prefix, currentPrefix );
```
The same here:
```suggestion
			return currentPrefix + prefix + stemmedWordWithoutPrefix + suffix;
```

hansjovis commented on packages/yoastseo/src/morphology/dutch/generateParticipleForm.js:
```diff
@@ -0,0 +1,153 @@
+	 * form so there is no need to create the participle form. */
+	if ( inseparablePrefix ) {
+		if ( suffix === "d" ) {
+			return addParticipleAffixes( stemmedWord, suffix );
```
```suggestion
			return stemmedWord + suffix;
```

agnieszkaszuba commented:
@hansjovis 
I have deleted the addParticipleAffixes function for Dutch entirely since it doesn't seem necessary according to your comments :)
Also, the addParticipleAffixes function that was put in the helpers is now moved back to its original file, the one for creating German participles. I had moved it to helpers because I thought it could be used for Dutch, but this turned out to not be the case.

hansjovis commented:
Acceptance test: ✅ .

I added some specs to test the code and committed them as well. Will wait until the Travis build passes before merging the branch.
